build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1414,6 +1414,10 @@ message ToolDetails {
 //
 // * name: `build.bazel.remote.execution.v2.requestmetadata-bin`
 // * contents: the base64 encoded binary `RequestMetadata` message.
+// Note: the gRPC library serializes binary headers encoded in base 64 by
```

Comment:
Maybe link to the gRPC spec where this is specified for headers ending in `-bin`?

Reply:
Done.